### DISCUSSION TITLE
Group Leader

### DISCUSSION THREAD

**Discussion by Zendaug** [graphql/graphiql#1933] [Ideas]
Hi folks, in Canvas, you can set a group leader for student teams. However, it doesn't seem to be possible to retrieve this information through Graphiql. Would it be possible to introduce this feature?

**Comment by acao** [comment]
I’m not sure what canvas is. They use graphiql for their schema it appears from your question, so probably best to ask them. Graphiql is used by many tools and platforms across languages but we aren’t involved with the design of third party schemas
